components/layout/HeroBanner.tsx

```diff
@@ -41,19 +41,18 @@ export default function HeroBanner({
 }: HeroBannerProps) {
   return (
     <Box sx={overrideStyles || heroStyles} textAlign="left">
-      <SectionContainer>
+      <SectionContainer sx={{}}>
```

Comment:
Do we need this empty object?

Reply:
Without this, have to remove py: 8 in heroStyles. But, then the index page would have a white section on top of the page instead of the purple margin. And if we change main tag on index page to <>, it would be resolved, but don't think it's a good solution to change main to <>. Have to look more to understand why the white section appears and how to fix it.